tests/rest_api/test_get_collection.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import pytest
+
+
+BASE_URL = "https://drb-api-qa.nypl.org"
+
+def test_get_collection():
+
+    url = BASE_URL + "/collection/3650664c-c8be-4d07-8d64-2d7003b02048"
+    expected_status_code = 200
+    response = requests.get(url)
+    assert response.status_code == expected_status_code, (
+        f"API call failed.\n"
+        f"Expected status code: {expected_status_code}\n"
+        f"Actual status code: {response.status_code}\n"
+        f"URL: {url}\n"
+        f"Response text: {response.text[:100]}..."
+    )
```

Comment:
Recommend using newlines for readability:

```
url = BASE_URL + "/collection/3650664c-c8be-4d07-8d64-2d7003b02048"

response = requests.get(url)
    
assert response.status_code == expected_status_code, (
        f"API call failed.\n"
        f"Expected status code: {expected_status_code}\n"
        f"Actual status code: {response.status_code}\n"
        f"URL: {url}\n"
        f"Response text: {response.text[:100]}..."
)
```

You don't necessarily need the newline on line 8.